packages/web/src/config.ts

```diff
@@ -20,9 +22,10 @@ import { browserMeta, pageMeta } from './metas';
 import { FetchTransport } from './transports';
 
 export interface BrowserConfig {
+  app: App;
+
   url?: string;
   apiKey?: string;
-  app: App;
   session?: Session;
   user?: User;
   globalObjectKey?: string;
```

Comment:
I think this object becomes a little harder to follow. We have metas, agent state flags (paused), configs etc.

Reply:
well, "advanced" stuff is optional :thinking:  what do you propose? 
If it's about the interface definition, I can maybe get clever and pull most of these from the core config interface

Reply:
I was thinking about child objects, similarly to what i proposed for the API (`pause` and `unpause` moved on transports etc.).

Reply:
I'm not a fan of nested config objects tbh, I think keeping it as flat as possible is most user friendly - much easier to discover what you need eg just by type suggestions